resources/assets/js/designer/components/modalList/modal-forms-list.vue

```diff
@@ -79,18 +79,21 @@
                 window.location.href = '/designer/' + this.processUid + '/form/' + data.uid;
             },
             onDelete(data, index) {
-                const CancelToken = ProcessMaker.apiClient.CancelToken;
-                ProcessMaker.apiClient
-                    .delete('process/' + this.processUid + '/form/' + data.uid,
-                        {
-                            cancelToken: new CancelToken(c => {
-                                this.cancelToken = c;
-                            })
-                        }
-                    )
-                    .then(response => {
-                        this.fetch();
-                    })
+                let that = this;
+                ProcessMaker.confirmModal('Caution!', '<b>Are you sure to delete </b>' + data.title + '?', '', function() {
+                    const CancelToken = ProcessMaker.apiClient.CancelToken;
+                    ProcessMaker.apiClient
+                        .delete('process/' + that.processUid + '/form/' + data.uid,
```

Comment:
This modal needs to be more generic. We will use to any time we delete anything.

Reply:
Not just for deleting processes

Reply:
the function can be used globally